nemo_curator/scripts/classifiers/quality_classifier_inference.py

```diff
@@ -29,7 +31,9 @@
 
 
 def main():
-    args = ArgumentHelper.parse_distributed_classifier_args().parse_args()
+    args = ArgumentHelper.parse_distributed_classifier_args(
```

Comment:
Instead of modifying every script to have this, can we just make it a prefix to all descriptions in the class?

Reply:
Done, thanks! Added to `ArgumentHelper`'s `__init__` function.